flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerMigrationTest.java

```diff
@@ -1,149 +0,0 @@
-/*
```

Comment:
We can't remove this because there is another PR that updates this for Flink 1.2 and 1.3: #4565